<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWSbg</class>
 <widget class="QWidget" name="QWSbg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWSbg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QTabWidget" name="statusWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_connection">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="SbgEcomLogStatus" name="sbgEcomLogStatus" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="SbgConnection" name="sbgConnection" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_data">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="SbgEcomLogData" name="sbgEcomLogData" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SbgConnection</class>
   <extends>QWidget</extends>
   <header>sbgconnection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SbgEcomLogStatus</class>
   <extends>QWidget</extends>
   <header>sbgecomlogstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SbgEcomLogData</class>
   <extends>QWidget</extends>
   <header>sbgecomlogdata.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
